File: examples/python-deps/hello.py
#!/usr/bin/env python3
"""Example demonstrating external Python package usage in Buck2.

This example uses the `click` package for CLI building.
To make this work with Buck2, you need a third-party setup.
"""

import click

@click.command()
@click.option("--name", default="World", help="Name to greet")
def main(name: str) -> None:
    click.echo(f"Hello, {name}!")

if __name__ == "__main__":
    main()

File: examples/typescript-hello-deps/hello.ts
// Example demonstrating external npm package usage in Buck2
//
// This example uses the `chalk` package for colored console output.
// To make this work with Buck2, you need npm/worker integration.

import chalk from 'chalk';

const name = process.argv[2] || 'World';
console.log(chalk.green(`Hello, ${name}!`));
